modules/images/dominant-color/class-wp-image-editor-gd-with-color.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * WordPress Image Editor Class for Image Manipulation through GD
+ * with dominant color detection
+ *
+ * @package performance-lab
+ * @since 1.0.0
+ */
+
+/**
+ * WordPress Image Editor Class for Image Manipulation through GD
+ * with dominant color detection
+ *
+ * @since 6.0.0
```

Comment:
This will not going WordPress 6.0.0. I would either make it performance plugin since, so 1.1.0 or 6.x for WP